gatsby-theme-patternfly-org/components/topNav.js

```diff
@@ -8,12 +8,23 @@ const TopNav = ({ location, context, navItems }) => {
     <Nav aria-label="Nav">
       <NavList variant="horizontal" className="ws-top-nav">
         {navItems.map(({ path, text, contexts }) => (
-          <NavItem
-            key={path}
-            isActive={location.pathname.includes(path) || (contexts || []).includes(context)}
-            >
-            <Link to={path}>{text}</Link>
-          </NavItem>
+          (path.includes('http'))
+            ? (
+              <NavItem
+                key={path}
+                isActive={location.pathname.includes(path) || (contexts || []).includes(context)}
+                >
+                <a href={path} className="ws-topnav-blog" target="_blank">{text}</a>
+              </NavItem>
+            )
+            : (
+              <NavItem
+                key={path}
+                isActive={location.pathname.includes(path) || (contexts || []).includes(context)}
+                >
+                <Link to={path}>{text}</Link>
```

Comment:
Move the ternary statement to inside the `NavItem` and make it `path.includes('//') ? <a href={path} target="_blank">{text}</a> : <Link to={path}>{text}</Link>`